draft-ietf-quic-multipath.md

```diff
@@ -1047,7 +1045,7 @@ it MUST close the connection with an error of type FRAME_ENCODING_ERROR.
 
 Receipt of multipath-specific frames
 that use a Path ID that is greater than the announced Maximum Paths value
-in the MAX_PATHS frame or in the initial_max_paths transport parameter,
+in the MAX_PATHS frame or in the initial_max_path_id transport parameter,
 if no MAX_PATHS frame was received yet,
 MUST be treated as a connection error of type MP_PROTOCOL_VIOLATION.
```

Comment:
To be consistent, we should we also change the name of that frame to MAX_PATH_ID and call the variable the "Maximum Path ID".

Reply:
That's true. I updated the Frame Name and Type from MAX_PATHS to MAX_PATH_ID, please review.

Reply:
Note that we only need to change Frame Type and Frame Name in this PR, because Martin has already changed the definition of Frame field "Maximum Path ID" in PR #372.